### PR TITLE
Show actions on hover/focus/active

### LINKED ISSUE #3048: Show app navigation actions menu on on hover and focus
Right now every app nav item with actions has a visible `...`. This shall be changed so only hovered and active elements have it.

Bildschirmfoto vom 2022-08-16 12-58-35

This will align the behavior with the *app content list* from Talk/Mail.

### PR DESCRIPTION
**Pull request by marcoambrosini**
fix #3048

Signed-off-by: Marco Ambrosini <marcoambrosini@icloud.com>

### CODE REVIEW

File: src/components/AppNavigationItem/AppNavigationItem.vue [hunk 1]
```diff
@@ -492,6 +519,67 @@ export default {
...
+		 *
+		 * @param {Event} e the keydown event
+		 */
+		handleTab(e) {
+			if (this.focused && this.hasActions) {
+				e.preventDefault()
+				this.$refs.actions.$refs.menuButton.$el.focus()
+				this.focused = false
+			} else {
+				console.log('blur')
```
Comment (ChristophWurst) [inline src/components/AppNavigationItem/AppNavigationItem.vue:579]: leftover?

**Comment by GretaD** [comment]
not sure if its related to this pr or not but the action menu is shown only when i hover middle left. Maybe it should be shown if i hover anywhere otherwise its not intuitive.


https://user-images.githubusercontent.com/12728974/185420016-839e520b-af0d-40ae-ae94-ca66b2adb4bc.mp4

**Comment by marcoambrosini** [comment]
This is not in a reviewable state yet :)

**Review by ChristophWurst — APPROVED** [review]
Code looks good

**Comment by GretaD** [comment]
theres still something wrong with the hover, on mail, when i hover over the number of unread messages the action menu is not shown. Maybe this could be fixed on apps separately? 


https://user-images.githubusercontent.com/12728974/185905280-52f1793f-f0e9-4b61-8f19-2ad8b19d3718.mp4

**Comment by GretaD** [comment]
or this other problem with showing the action menu on account settings: 
https://user-images.githubusercontent.com/12728974/185905827-db08ab72-723c-4bf5-974f-33c3175a823d.mp4

**Comment by marcoambrosini** [comment]
Are you sure you pulled @GretaD?

**Comment by GretaD** [comment]
> Are you sure you pulled @GretaD?

yes, but i think the linking failed me, im working on another ticket and the linking is not working properly, will try again in a minute

**Comment by skjnldsv** [comment]
> yes, but i think the linking failed me, im working on another ticket and the linking is not working properly, will try again in a minute

Have you tried again @GretaD ?

**Review by skjnldsv — APPROVED** [review]

**Comment by raimund-schluessler** [comment]
> theres still something wrong with the hover, on mail, when i hover over the number of unread messages the action menu is not shown. Maybe this could be fixed on apps separately?

The same happens for Tasks. The counter disappears on hover, but the actions don't appear. They are only visible when the entry is selected *and* hovered. Do we need to adjust something in the apps? @marcoambrosini

**Comment by marcoambrosini** [comment]
I can't reproduce this in mail, it seems to be working just fine there. Are you using some custom css? 

https://user-images.githubusercontent.com/26852655/186347916-37b8c782-a0cd-4804-8d0a-92e487ac6812.mov

**Comment by GretaD** [comment]
> I can't reproduce this in mail, it seems to be working just fine there. Are you using some custom css?

after the update i cannot reproduce it either, the hover looks fine

**Pull request merged**